app/code/Magento/Indexer/Console/Command/IndexerStatusMviewCommand.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Copyright © 2013-2017 Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Indexer\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+use Magento\Framework\Mview\View;
+
+/**
+ * Command for displaying status of mview indexers.
+ */
+class IndexerStatusMviewCommand extends Command
+{
+    /** @var \Magento\Framework\Mview\View\CollectionInterface $mviewIndexersCollection */
+    private $mviewIndexersCollection;
+
+    public function __construct(
+        \Magento\Framework\Mview\View\CollectionInterface $collection
+    ) {
+        $this->mviewIndexersCollection = $collection;
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName('indexer:status:mview')
+            ->setDescription('Shows status of Mview Indexers and their queue status');
+
+        parent::configure();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $table = $this->getHelperSet()->get('table');
+            $table->setHeaders(['ID', 'Mode', 'Status', 'Updated', 'Version ID', 'Backlog']);
+
+            $rows = [];
+
+            /** @var \Magento\Framework\Mview\View $indexer */
+            foreach ($this->mviewIndexersCollection as $indexer) {
```

Comment:
Actually it's not indexer, it's view. So it's better to use one name for this object type